src/ui/window.c

```diff
@@ -1335,19 +1335,23 @@ win_show_status_string(ProfWin* window, const char* const from,
     win_appendln(window, presence_colour, "");
 }
 
-static void
+/** Corrects the visual representation of a message with prior check for sender validity.
+ *
+ * @returns TRUE if the message was successfully corrected and should not be printed, FALSE otherwise
```

Comment:
Still this doxygen `@returns`. But not important I can change that after merging.

Reply:
> Still this doxygen `@returns`. But not important I can change that after merging.

it's doxygen style, but it doesn't take excessive space, but at the same time, provide better readability